packages/web/src/scss/tools/_dictionaries.scss

```diff
@@ -145,6 +147,46 @@
     }
 }
 
+// Mixin to generate placement classes based on a dictionary
```

Comment:
Has CSS/SCSS a documenting standard like JSDoc or any Doc Blocks? E.g.
```
/**
 * @param ...
 * @return ...
 */
```

I think it would be great not to reinvent the wheel. :-)

Reply:
Nothing I'm aware of. Currently I'm using the same format as in our other dictionary mixins and functions.

But I like the idea! Please feel free to drop it onto our discussion board, or even file an issue right away. If we choose a format like that, I'd like to see it in all our Sass files.

cc @crishpeen